edge-hub/src/Microsoft.Azure.Devices.Edge.Hub.Service/Startup.cs

```diff
@@ -47,6 +48,12 @@ public IServiceProvider ConfigureServices(IServiceCollection services)
 
         public void Configure(IApplicationBuilder app)
         {
+            // Separate metrics endpoint from https authentication
+            app.Map("/metrics", metricsApp =>
+            {
+                metricsApp.UseMetricServer(string.Empty);
```

Comment:
From the stub in Prometheus.AspNetCore I can see string.empty maps to this parameter:
```
[NullableAttribute(2)] string? url = "/metrics"
```

Is your intention here to let it take the default /metrics? If so, your arg isn't null so won't this not work? Or will string.empty be treated the same as a null?

Reply:
if you want to go for default, you can just call UseMetricServer() without arguments.

Reply:
It needs to be empty string since `app.Map("/metrics", metricsApp =>` already routes from /metrics. If I passed in null, it would listen to `edgehub:9600/metrics/metrics`. Whatever I pass in there will be appended to `edgehub:9600/metrics`, so I "append" empty string.